### PR TITLE
About adjust

### PR DESCRIPTION
**Pull request by lala-laura**
Pull request for About page

### CODE REVIEW

File: about.html [hunk 1]
```diff
@@ -5,72 +5,78 @@
...
+            <p>
+                Established on Earth in the year 2021, <span class="shoutout">Who Travels</span> started small, with only trips to past and present Earth locations and Mars. Now with access to nearly every Earth era and over 60% of the Galaxy, <span class="shoutout">Who Travels</span> is the largest (and only!) Time and Space Travel Company on any planet!
+            </p>
 
-            <p>Who Travels offers instantaneous trips that will have you home before breakfast or overnight transit with comfortable accommodations. Overnight excursions include meals from our gourmet food machine, spacious lounging rooms, a swimming pool, a conservatory, and a personal view from the control room. (rotating image of interior?)</p>
+            <p><span class="shoutout">Who Travels</span> offers instantaneous trips that will have you home before breakfast or overnight transit with comfortable accommodations. Overnight excursions include meals from our gourmet food machine, spacious lounging rooms, a swimming pool, a conservatory, and a personal view from the control room. (rotating image of interior?)</p>
```
Comment (loganmorganml1) [inline about.html:66]: I don't know how I feel with the styling of the "shoutout" section -- I think just having it bolded would work and look a bit less clunky.

**Review by loganmorganml1 — CHANGES_REQUESTED** [review]
One small styling change, but looks good!

File: about.html (hunk 1 above)
Comment (lala-laura) [reply about.html:66]: I think I agree, I changed it to bold

**Review by loganmorganml1 — APPROVED** [review]
Looks good, approved!

**Pull request merged**